### DISCUSSION TITLE
Welcome to Todoser Discussions!

### DISCUSSION THREAD

**Discussion by XaoGao** [XaoGao/Todoser#59] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by XaoGao** [comment]
@vovs03  @sas2job  @RaptorialThing  @mrjoshua22  Если возникнут идеи, что реализовать или какие технологии хотите использовать то пиште сюда

**Reply by vovs03** [reply]
Я бы хотел заюзать `Makefile`, чтобы не вспоминать все команды, как они пишутся, особенно длинные.

**Reply by vovs03** [reply]
Ещё прикрутить хуки на прекоммит,
- [ ]  линтер
- [ ] spellcheck
